docs/recipes/debugging-with-webstorm.md

```diff
@@ -13,6 +13,8 @@ In the `JavaScript file` field specify the path to AVA in the project's `node_mo
 
 In the `Application parameters` pass the CLI flags you're using and the test files you would like to debug, for example `--verbose test.js`.
 
+In the `Node parameters` pass the `--inspect-brk` flag to enable the Node inspector or `--inspect-brk=port` if you want to listen it on a specific port.
```

Comment:
@develar Thanks for pointing out the node version. I updated my statement to include a solution for both 7+ and previous versions.

I also removed the statement about `--inspect-brk=port`. There are other flags that can also be used and I thought it is not the place to mention all of them. `--inspect-brk` is enough to make it work and if someone wants to try different things than they should look at Node.js documentation.